daemon/system-info.sh

```diff
@@ -189,4 +286,11 @@ echo "NETDATA_SYSTEM_VIRTUALIZATION=${VIRTUALIZATION}"
 echo "NETDATA_SYSTEM_VIRT_DETECTION=${VIRT_DETECTION}"
 echo "NETDATA_SYSTEM_CONTAINER=${CONTAINER}"
 echo "NETDATA_SYSTEM_CONTAINER_DETECTION=${CONT_DETECTION}"
+echo "NETDATA_CPU_LOGICAL_CPU_COUNT=${LCPU_COUNT}"
+echo "NETDATA_CPU_VENDOR=\"${CPU_VENDOR}\""
```

Comment:
Please strip the extra internal quotes so that it is consistent with the other cases. When Netdata parses the output of the script it takes everything between the `=` and the newline as a string, we don't break on spaces.

Reply:
Ah, OK, I thought we were parsing this as shell variables. I'll update that shortly.